Service/Inertia.php

```diff
@@ -156,7 +156,7 @@ public function render($component, $props = [], $viewData = [], $context = []):
             });
 
         array_walk_recursive($props, function (&$prop) {
-            if (is_callable($prop) || $prop instanceof LazyProp) {
+            if (!is_string($prop) && $prop instanceof LazyProp) {
```

Comment:
Why is this `!is_string` check necessary?  If it's an instanceof `LazyProp` will it ever be a string?

Reply:
already removed the is_string